tests/uki_test.go

```diff
@@ -256,5 +269,33 @@ var _ = Describe("kairos UKI test", Label("uki"), Ordered, func() {
 		out, err = vm.Sudo("ls /usr/local/after-reset-file")
 		Expect(err).ToNot(HaveOccurred(), out)
 		Expect(out).ToNot(MatchRegexp("No such file or directory"))
+
+		By("upgrading a single boot entry")
+		upgradeImage := os.Getenv("UPGRADE_IMAGE")
+		out, err = vm.Sudo(fmt.Sprintf("kairos-agent --debug upgrade --source oci:%s --boot-entry %s", upgradeImage, os.Getenv("EXPECTED_SINGLE_ENTRY")))
```

Comment:
This is strange. With this missing: https://github.com/kairos-io/kairos-agent/pull/499/files#r1734786839
this should have never worked :thinking: .

Reply:
all good (see the comment on the link)